frontend/src/components/agents-table/agent-form-modal.tsx

```diff
@@ -116,6 +118,22 @@ export function AgentFormModal({
               )}
             </div>
             
+            <div className="flex flex-col gap-1">
+              <Label htmlFor="agent-description">Description</Label>
+              <textarea 
```

Comment:
Do we have a TextArea component in the ui/ directory already? It would make the styling easier and more consistent with the rest of the application